pkg/quota/admission/clusterresourceoverride/admission.go

```diff
@@ -166,48 +166,36 @@ func (a *clusterResourceOverridePlugin) Admit(attr admission.Attributes) error {
 	for _, container := range pod.Spec.Containers {
 		resources := container.Resources
 		memLimit, memFound := resources.Limits[kapi.ResourceMemory]
-		if memFound && a.config.memoryRequestToLimitRatio.Cmp(zeroDec) != 0 {
+		if memFound && a.config.memoryRequestToLimitRatio != 0 {
 			// memory is measured in whole bytes.
 			// the plugin rounds down to the nearest MiB rather than bytes to improve ease of use for end-users.
-			amount := multiply(memLimit.AsDec(), a.config.memoryRequestToLimitRatio)
-			roundDownToNearestMi := multiply(divide(amount, miDec, 0, inf.RoundDown), miDec)
-			value := resource.Quantity{Amount: roundDownToNearestMi, Format: resource.BinarySI}
-			if memFloor.Cmp(value) > 0 {
-				value = *(memFloor.Copy())
+			amount := float64(memLimit.Value()) * a.config.memoryRequestToLimitRatio
+			q := resource.NewQuantity(int64(amount), memLimit.Format)
+			q.RoundUp(resource.Mega)
```

Comment:
RoundUp forces the minimum value to be above 1M and increases
precision.  In this case you'd want to round 1.4 to 2